include/Graphs/CFLGraph.h

```diff
@@ -45,12 +46,29 @@ typedef GenericEdge<CFLNode> GenericCFLEdgeTy;
 
 class CFLEdge: public GenericCFLEdgeTy
 {
+    static u32_t offset;
+
 public:
     CFLEdge(CFLNode *s, CFLNode *d, GEdgeFlag k = 0):
         GenericCFLEdgeTy(s,d,k)
     {
     }
     ~CFLEdge() override = default;
+
+    inline GEdgeKind getEdgeKind() const
+    {
+        return this->getEdgeKindWithoutMask();
+    }
+
+    inline static GEdgeKind getAttributedKind(GEdgeFlag attribute, GEdgeFlag flag)
+    {     
+        return ((attribute << CFLEdge::EdgeKindMaskBits)| flag );
+    }
+
+    inline static GEdgeKind getNormalKind(GEdgeFlag attributedKind)
+    {
+        return (CFLEdge::EdgeKindMask & attributedKind);
+    }
```

Comment:
remove these since we have CFLGrammar ones (they are static and can be used globally)